resources/views/container/layouts/metric.blade.php

```diff
@@ -8,12 +8,16 @@
                 <small class="text-muted block mb-1">{{ __($key) }}</small>
                 <p class="h4 mb-1 text-black font-thin">{{ $metric['keyValue'] }}</p>
 
-                @if((float)$metric['keyDiff'] < 0)
-                    <small class="text-danger">{{ $metric['keyDiff'] }} % <i class="icon-arrow-down"></i></small>
-                @else
-                    <small class="text-success">{{ $metric['keyDiff'] }} % <i class="icon-arrow-up"></i></small>
+                @if(isset($metric['keyDiff']))
+                    @if((float)$metric['keyDiff'] < 0)
+                        <small class="text-danger">{{ $metric['keyDiff'] }} % <i class="icon-arrow-down"></i></small>
+                    @elseif((float)$metric['keyDiff'] == 0)
+                        <small class="text-muted">{{ $metric['keyDiff'] }} % <i class="icon-refresh"></i></small>
```

Comment:
Not sure if we need an icon to indicate zero.  I copied this layout from the Yandex.Metrica service and there was zero as positive.  I have nothing against it

Reply:
I think this is good to indicate that the data not changed.

Reply:
Probably you did not understand me. I meant exactly the `refresh icon`